README.rst

```diff
@@ -165,6 +183,24 @@ Since we only want to change the behaviour of these community submission request
     REQUESTS_PERMISSION_POLICY = CurationRDMRequestsPermissionPolicy
 
 
+Permit the moderators to view the draft under review
+----------------------------------------------------
+
+For curation reviews to make sense, it is of course vital for the moderators to be able to view the drafts in question.
+Per default, `Invenio-Curations` will create access grants for users with the moderation role as part of the curation requests.
+This should work out of the box without any further configuration needed.
+However, it has the downside of creating additional artifacts in the system, and users could accidentally revoke access for moderators by revoking this access grant.
```

Comment:
To be honest, I completely forgot about the second part. That is actually a huge issue. We should either disallow to remove the curation group from the access or switch to permission based completely

Reply:
Either way is fine with me, but perhaps the first option would be simpler for users.
At TU Wien, we have customized our permission policy to death and frankly I'm scared every time I have to touch it.